examples/feature_interface/partcad.yaml

```
@@ -36,7 +36,7 @@
     abstract: True
     ports:
       m3:
-        location: [[0, 0, 0], [0, 0, 1], 0] # redandant, for demonstration
+        location: [[0, 0, 0], [0, 0, 1], 0] # redundant, for demonstration
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)